core/trino-main/src/test/java/io/trino/sql/routine/TestSqlFunctions.java

```diff
@@ -339,6 +339,30 @@ FUNCTION test_repeat_continue()
         assertFunction(sql, handle -> assertThat(handle.invoke()).isEqualTo(7L));
     }
 
+    @Test
+    void testLoop()
```

Comment:
did you mean `add one for LOOP` in commit message?

Reply:
thanks, fixed